corehq/apps/events/tests/test_tasks.py

```diff
@@ -65,22 +66,22 @@ def test_cases_created_for_mobile_workers(self):
         """
         user_id_case_mapping = get_existing_cases_by_user_ids(self.domain)
         user_ids = list(user_id_case_mapping.keys())
-        sync_mobile_worker_attendees(domain_name=self.domain, user_id=self.webuser.user_id)
+        tasks.sync_mobile_worker_attendees(domain_name=self.domain, user_id=self.webuser.user_id)
```

Comment:
I'm curious as to how celery tasks are handled in tests. Do they still work?

Reply:
Ah, so celery is not used here. To push this task to a queue, you'd normally call it like `tasks.sync_mobile_worker_attendees.delay(...)`, but here we're only calling the function itself, instead of the delay method on the function, which then calls the function itself.

Reply:
Ah got it, that makes sense. Thanks for clarifying!